packages/storybook/src/Tab.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { RenderOptions } from '@storybook/addons';
+import { useStorybookState, useStorybookApi } from '@storybook/api';
+import { config } from './config';
+import { AddonPanel } from '@storybook/components';
+
+const style = {
+  display: 'block',
+  height: '90vh',
+  width: '100%',
+};
+
+export const Tab = (props: RenderOptions) => {
+  const state = useStorybookState();
+  const api = useStorybookApi()
+  requestAnimationFrame(() => api.toggleFullscreen(props.active))
+  const storyId = state.storyId;
+  React.useEffect(() => {
+    const script = document.getElementById(config.addonId);
+    if (script) {
+      return;
+    }
+    const builderScript = document.createElement('script');
+    builderScript.src = 'https://cdn-qa.builder.io/js/editor';
+    builderScript.id = config.addonId;
+    document.body.appendChild(builderScript);
+  });
+
+  return (
+    <AddonPanel {...props}>
+      {storyId ? (
+        <builder-editor
+          style={style}
+          options={`{
+        "floatingLeftSidebar": false,
+        "rtlMode": false,
```

Comment:
`rtlMode` and `floadtingLeftSidebar` isn't needed here

something I think we should do tho is add a couple options (or one simple `storybookMode`) option where we can do some handy things like allowing saving pages made to files and/or react code, etc